airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/StandardSyncPersistence.java

```diff
@@ -187,6 +187,7 @@ private void writeStandardSync(final StandardSync standardSync, final DSLContext
           .set(CONNECTION.DESTINATION_ID, standardSync.getDestinationId())
           .set(CONNECTION.NAME, standardSync.getName())
           .set(CONNECTION.CATALOG, JSONB.valueOf(Jsons.serialize(standardSync.getCatalog())))
+          .set(CONNECTION.FIELD_SELECTION_DATA, JSONB.valueOf(Jsons.serialize(standardSync.getFieldSelectionData())))
```

Comment:
will we need to add this for connection create as well? not sure of the UX here

Reply:
Good call, not sure how I missed this - added.